### PR TITLE
Feature/common data import task

### PR DESCRIPTION
**Pull request by flibbertigibbet**
Rake tasks to support #43 (A).
- Import common data into the local common data account.
- Remove table quota limit (for local common data account)
- Also updates uncompressed JS (result of #51)

### CODE REVIEW

File: lib/tasks/setup.rake [hunk 1]
```diff
@@ -168,5 +168,23 @@ DESC
...
+    task :remove_table_quota => :environment do
+      raise "Set USER_NAME environment variable" if ENV['USER_NAME'].blank?
+      users = ::User.filter(:username => ENV['USER_NAME']).all
+      if users.empty?
+        raise "User doesn't exist"
+      else
+        u = users.first
+        u.update(:table_quota => nil)
+        if !u.save
+          rais u.errors.inspect
```
Comment (CloudNiner) [inline lib/tasks/setup.rake]: Typo: `rais` -> `raise`

**Comment by CloudNiner** [comment]
+1

**Comment by CloudNiner** [comment]
If you're planning on using these in the azavea/vagrant-cartodb repo, be wary. That repo is intended to support an install of the upstream cartodb repo so if you use these functions there, we'll need some kind of workaround or alternate strategy.

**Comment by flibbertigibbet** [comment]
I've made it something to enable via a vagrant group variable, as we had discussed previously.

File: lib/tasks/setup.rake (hunk 1 above)
Comment (flibbertigibbet) [inline lib/tasks/setup.rake]: Same typo exists in `change_user_password` rake task (where this user fetch/error handle came from), will fix there as well.

**Comment by CloudNiner** [comment]
Right, enabling local common data as a flag makes sense. But if we're going to provide it to the general community as an option, then it should work for the default case, which is using the upstream cartodb repository. As it stands now, this will fail to run because the rake task can't be found and that's not a great experience for someone using the vagrant-cartodb repo for the first time. I figured something like a static sql file that could be imported via pgsql during provisioning would be a good first step. This solution is way more flexible than that though, which is great.

These functions are pretty generic and could be made even more so, so they could be useful to upstream cartodb. I wonder if they would accept a pull request for these, as that would solve our problem. Or maybe the cartodb folks have an already existing alternative for copying the data library datasets to another account? @javisantana any thoughts?

**Comment by flibbertigibbet** [comment]
A static SQL import files for test data sets? I imagine that would be unwieldy for putting in a git repo.

**Comment by CloudNiner** [comment]
Yeah, that was a quick brainstorm, not necessarily a great idea. As discussed offline, could also just configure the local cartodb instance with the user account and app_config to pull data from it, and leave it up to the user to manually import a few datasets to use. That would be a bit more flexible, but requires the user of the dev env to do something manually after provision. Other danger of running the rake task is that it pulls a ton of data, when the local instance probably only needs a few small datasets in development.

Barring other suggestions from @javisantana, maybe we remove (or disable with a flag) the rake tasks for now, so that this common data set up can be merged and we can continue forward?

**Comment by flibbertigibbet** [comment]
Import is separately configurable flag in azavea/vagrant-cartodb#10. Reloads common data for all users after import, which is necessary for it to show in UI.

**Comment by CloudNiner** [comment]
LGTM

**Comment by flibbertigibbet** [comment]
Merging as-is, though the common data loader does not respect non-default ports.

**Pull request merged**